fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/mappers/SketchRoundUpMapper.java

```diff
@@ -38,12 +38,15 @@ public SketchRoundUpMapper(String columnName) {
     }
 
     @Override
-    protected Result map(Result result, Schema schema) {
+    protected Result map(Result result, ResultSetSchema schema) {
         if (columnName == null) {
             throw new IllegalStateException("Cannot map results without a column name");
         }
 
-        MetricColumn metricColumn = (MetricColumn) schema.getColumn(columnName);
+        MetricColumn metricColumn = schema.<MetricColumn>getColumn(columnName, MetricColumn.class).orElseThrow(
+                () -> new IllegalStateException("Unexpected missing column")
```

Comment:
Next commit will add capture for ResultSetMapper errors above where mapping is called.